src/Components/blazor/src/Microsoft.AspNetCore.Blazor/Hosting/BlazorWebAssemblyHost.cs

```diff
@@ -4,12 +4,12 @@
 namespace Microsoft.AspNetCore.Blazor.Hosting
 {
     /// <summary>
-    /// Used to to create instances a Blazor host builder for a Browser application.
+    /// Used to create an instances of Blazor host builder for a Browser application.
```

Comment:
```suggestion
    /// Used to create an instance of Blazor host builder for a Browser application.
```

As long as we're fixing grammar, instances => instance :)